benchmarks/benchmarks/stats.py

```diff
@@ -92,5 +92,15 @@ def time_distribution(self, distribution, properties):
                 stats.beta.rvs(size=1000, a=5, b=3, loc=4, scale=10)
             elif properties == 'fit':
                 stats.beta.fit(self.x, a=5, b=3, loc=4, scale=10)
-        
+
+
+class DescriptiveStats(Benchmark):
+    def setup(self):
+        np.random.seed(12345678)
+        self.many_levels = np.random.randint(1000, size=(1000, 10))
```

Comment:
Pretty sure the random seed will be consumed on the first random call -- instead of adding a second fixed seed, probably easier to just parametrize the data sizes and keep a single fixed seed and a single i.e., self.levels type call instead of two separate ones.